src/main/java/io/fabric8/vertx/maven/plugin/mojos/RunMojo.java

```diff
@@ -33,8 +36,111 @@
     requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
 public class RunMojo extends AbstractRunMojo {
 
+    /**
```

Comment:
I'm not sure of the approach here. I would have preferred a simple <args> such as:

```
<args>
  <arg>--cluster</arg>
  <arg>--instances</arg>
  <arg>2</arg>
</args>
```

Like the exec maven plugin.